plugins/legacy-preset-chart-nvd3/src/NVD3Vis.js

```diff
@@ -418,10 +425,22 @@ function nvd3Vis(element, props) {
           chart.labelType(pieLabelType);
         } else if (pieLabelType === 'key_value') {
           chart.labelType(d => `${d.data.x}: ${numberFormatter(d.data.y)}`);
-        } else if (pieLabelType === 'key_percent') {
+        } else {
+          // pieLabelType in ['key_percent', 'key_value_percent']
           const total = d3.sum(data, d => d.y);
-          chart.tooltip.valueFormatter(d => `${((d / total) * 100).toFixed()}%`);
-          chart.labelType(d => `${d.data.x}: ${((d.data.y / total) * 100).toFixed()}%`);
+          const getPercent = function (d) {
+            return `${((d / total) * 100).toFixed()}%`;
+          };
```

Comment:
Can you refactor this with `getNumberFormatter(NumberFormats.PERCENT_2_POINT)`?

Reply:
done